extra_data/tests/test_reader_mockdata.py

```diff
@@ -679,6 +680,16 @@ def test_open_file(mock_sa3_control_data):
         assert 'METADATA/dataSources/dataSourceId' in file_access.file
 
 
+def open_run_daemonized_helper(path):
+    RunDirectory(path, parallelize=False)
+
+def test_open_run_daemonized(mock_fxe_raw_run):
+    p = Process(target=open_run_daemonized_helper, args=(mock_fxe_raw_run,), daemon=True)
+    p.start()
+    p.join()
+
+    assert p.exitcode == 0
```

Comment:
You can maybe add the test that fails when the flag is set, for completeness?

Else LGTM

Reply:
I don't think it's so important to test that something fails :wink: . But I think it would be good to have a brief explanatory comment on this test, to make it clear what we care about once it doesn't have the context of this PR.